### PR TITLE
fix: Val Exits

### PR DESCRIPTION
**Pull request by davidterpay**

### CODE REVIEW

File: abci/strategies/aggregator/price_applier.go [hunk 1]
```diff
@@ -78,6 +78,12 @@ func (opa *oraclePriceApplier) ApplyPricesFromVoteExtensions(ctx sdk.Context, re
 		return nil, err
 	}
 
+	opa.logger.Info(
```
Comment (Eric-Warehime) [inline abci/strategies/aggregator/price_applier.go]: Debug pls

File: abci/ve/utils.go [hunk 2]
```diff
@@ -208,8 +213,6 @@ func ValidateVoteExtensions(
 		if !cmtPubKey.VerifySignature(extSignBytes, vote.ExtensionSignature) {
 			return fmt.Errorf("failed to verify validator %X vote extension signature", valConsAddr)
 		}
-
-		sumVP += vote.Validator.Power
```
Comment (Eric-Warehime) [inline abci/ve/utils.go:212]: ~~We need this.~~

Comment (davidterpay) [reply abci/ve/utils.go:212]: kek

File: abci/ve/utils.go [hunk 3]
```diff
@@ -182,10 +182,15 @@ func ValidateVoteExtensions(
...
+		// If the validator does not have a valid public key, we skip the signature verification logic but still include
+		// the validator's voting power in the total voting power. The app may have pruned the validator's public key
+		// from the store, but comet considered the validator as active and included them in the commit since there
+		// is a 1 block delay between the validator set update on the app and comet.
+		sumVP += vote.Validator.Power
 		valConsAddr := sdk.ConsAddress(vote.Validator.Address)
 		pubKeyProto, err := valStore.GetPubKeyByConsAddr(ctx, valConsAddr)
 		if err != nil {
-			return fmt.Errorf("failed to get validator %X public key: %w", valConsAddr, err)
+			continue
```
Comment (g3ndo9kar9) [inline abci/ve/utils.go:193]: can we check that this is a DNE error before continuing?

File: tests/integration/slinky_validator_suite.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+		}
+
+		oracle := GetOracleSideCar(node)
+		SetOracleConfigsOnOracle(oracle, oracleConfig)
+		s.Require().NoError(RestartOracle(node))
+	}
+
+	// wait for the oracle to start
+	s.T().Log("Waiting for oracle to start...")
+	time.Sleep(45 * time.Second)
```
Comment (g3ndo9kar9) [inline tests/integration/slinky_validator_suite.go:84]: why do we have a sleep here? Is there no way to block on whether a price is included in these VEs?

**Review by Eric-Warehime — APPROVED** [review]

File: abci/ve/utils.go (hunk 3 above)
Comment (technicallyty) [inline abci/ve/utils.go:193]: should we have a comment on why we made this change?

Comment (davidterpay) [reply abci/ve/utils.go:193]: I think the comment above should cover it. do u think there is anything else i should add to the comment?

Comment (technicallyty) [reply abci/ve/utils.go:193]: oh whoops, i missed that lol 😪😪 😪😪

**Review by g3ndo9kar9 — APPROVED** [review]
lgtm

**Pull request merged**